janitor/functions/select_columns.py

```diff
@@ -26,26 +27,18 @@ def select_columns(
 
     Optional ability to invert selection of columns available as well.
 
+    Example:
 
-    Functional usage example:
-
-    ```python
-       import pandas as pd
-       import janitor as jn
-
-       df = pd.DataFrame(...)
-
-       df = jn.select_columns('a', 'b', 'col_*',
-                              invert=True)
-    ```
-
-    Method-chaining example:
-
-    ```python
-        df = (pd.DataFrame(...)
-              .select_columns('a', 'b', 'col_*',
-              invert=True))
-    ```
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.DataFrame({"col1": [1, 2], "foo": [3, 4], "col2": [5, 6]}); df
```

Comment:
can we shift the `df` after the `;` down to a new line. looks cleaner IMO

Reply:
Appreciate the attention to detail here, @samukweku!